backend/api/views/main.py

```diff
@@ -52,29 +52,37 @@ def create_person():
     )
 
 
-"""
+'''
 function that is called when you visit /response_rates
 Parameters
-    date: date string with the format MMDDYYYY
     year: year string with format YY
     optional tract_id: 11-digit tract id string
     optional state: two digit id string
-Either date or year is required.
-"""
-
-
-@main.route("/response_rates", methods=["GET"])
+'''
+@main.route("/rate", methods=["GET"])
 def get_response_rates():
     responses_rate = None
 
+    year = request.args.get("year", None)
     tract_id = request.args.get("tract_id", None)
-    date = request.args.get("date", None)
+    state = request.args.get("state", None)
+
+    if year:
+        response_rates = get_last_response_rates_by_year(year, tract_id, state)
+    else:
+        return create_response(status=422, message="Missing request parameters")
+
+    return create_response(data={"response_rates": response_rates})
+
+@main.route("/rates_per_period", methods=["GET"])
+def get_response_rates_per_period():
```

Comment:
could you merge master to resolve conflicts so we're not reviewing the backend changes twice?